frontend/src/components/RichEditor.js

```diff
@@ -29,24 +31,41 @@ import { getEditorState } from '../utils';
 const RichEditor = ({
   ariaLabel, value, onChange,
 }) => {
+  const [height, setHeight] = useState(BASE_EDITOR_HEIGHT);
+
+  const editorRef = useRef();
+
   let defaultEditorState;
   if (value) {
     defaultEditorState = getEditorState(value);
   }
 
   const onInternalChange = (currentContentState) => {
+    // an improvement would be converting to rems to match the initial height but
+    // it might not matter since we're deriving the scroll height directly from the client here
+    setHeight(editorRef.current && editorRef.current.scrollHeight ? `${editorRef.current.scrollHeight}px` : BASE_EDITOR_HEIGHT);
```

Comment:
Is there every going to be a case here where the 'editorRef.current.scrollHeight' would be set to rem? Just curious if it would mess with the hardcoded 'px'.